docs/Stdlib/python/Tutorials/data_processing.md

```diff
@@ -1,78 +1,314 @@
+new_row = Row([1, 2, 3], TableSchema(
+    {
+        "column1": IntColumnType(),
+        "column2": IntColumnType(),
+        "column3": IntColumnType()
+    }
+))
 
-    # if no lambda is provided, the columns will be sorted alphabetically
-    table = table.sort_columns()
-    ```
+existing_column = table.get_column("column2")
 
-1. Filter rows with a given query
+existing_row = table.get_row(2)
+```
 
-    ```python
-    # returns a new table where each row has the value 1 in the column "NameOfColumn1"
-    table = table.filter_rows(lambda row: row.get_value("NameOfColumn1") == 1)
-    ```
+You can also create a Table from a list of [Rows][safeds.data.tabular.Table.from_rows] or [Columns][safeds.data.tabular.Table.from_columns] and get a list of [Rows][safeds.data.tabular.Table.to_rows] or [Columns][safeds.data.tabular.Table.to_columns] from an existing Table.
 
-1. Write data to a CSV or JSON file
+```python
+rows = table.to_rows()
+columns = table.to_columns()
 
-    ```python
-    # write to a .csv-file
-    table.to_csv("path_to_csv_output_file.csv")
+rows.append(new_row)
+table_new_row = Table.from_rows(rows)
 
-    # write to a .json-file
-    table.to_json("path_to_json_output_file.json")
-    ```
+columns.append(new_column)
+table_new_column = Table.from_columns(columns)
+```
+
+The output of table_new_row is:
+
+| column1 | column2 | column3 |
+| ------- | ------- | ------- |
+| 3       | 2       | 1       |
+| 4       | 2       | 1       |
+| 8       | 1       | 1       |
+| 6       | 6       | 1       |
+| 5       | 3       | 1       |
+| 1       | 2       | 3       |
+
+And the output of table_new_column is:
+
+| column1 | column2 | column3 | NewColumn |
+| ------- | ------- | ------- | --------- |
+| 3       | 2       | 1       | 1         |
+| 4       | 2       | 1       | 2         |
+| 8       | 1       | 1       | 3         |
+| 6       | 6       | 1       | 4         |
+| 5       | 3       | 1       | 5         |
+
+## Add Rows and Columns to a Table
+
+You might think this is a very laborious approach to add a column or a row to a Table. You can also add a new [Column][safeds.data.tabular.Table.add_column] or [Row][safeds.data.tabular.Table.add_row] to an existing Table.
+
+```python
+table_new_row_2 = table.add_row(new_row)
+
+table_new_column_2 = table.add_column(new_column)
+```
+
+## Drop and Keep Columns from a Table
+
+The outputs are the same tables you created in the example above.
+
+Sometimes it might be useful to remove some columns. Therefore, you can create a new table which will either [keep][safeds.data.tabular.Table.keep_columns] or [drop][safeds.data.tabular.Table.drop_columns] some columns from the original table.
+
+```python
+table_without_column_1 = table.drop_columns(["column1"])
+
+table_with_only_column_1_and_2 = table.keep_columns(["column1", "column2"])
+```
+
+First the table dropped column1. This leaves a new Table with column2 and column3:
```

Comment:
```suggestion
First the column1 is dropped. This leaves a new table containing column2 and column3:
```

Reply:
```suggestion
First the column1 is dropped. This leaves a new Table containing column2 and column3:
```
A new instance of the Table class is created, therefore i would keep the capitalization

Reply:
```suggestion
First column1 is dropped. This leaves a new Table containing column2 and column3:
```